File: Tula/Bars.GkhGji.Regions.Tula/libs/B4/model/ActSurvey.js
/*
В Сахе сущность перекрвается с новыми полями
*/
Ext.define('B4.model.ActSurvey', {
    extend: 'B4.model.DocumentGji',
    idProperty: 'Id',
    
    proxy: {
        type: 'b4proxy',
        controllerName: 'ActSurvey'
    },
    fields: [
        { name: 'Inspection', defaultValue: null },
        { name: 'Area' },
        { name: 'Flat' },
        { name: 'Reason' },
        { name: 'Description' },
        { name: 'FactSurveyed', defaultValue: 10 },
        { name: 'RealityObjectsList' },
        { name: 'ParentDocumentsList' },
        { name: 'TypeDocumentGji', defaultValue: 40 },
        { name: 'DocumentDate' },
        { name: 'DocumentNumber' },
        { name: 'State', defaultValue: null },
        { name: 'MunicipalityNames' },
        { name: 'RealityObjectAddresses' },
        { name: 'InspectorNames' },
        { name: 'InspectionId', defaultValue: null },
        { name: 'TypeBase', defaultValue: null },
        { name: 'DateOf' },
        { name: 'TimeStart' },
        { name: 'TimeEnd' },
        { name: 'ConclusionIssued', defaultValue: 20 }
    ]
});

File: Bars.GkhGJI/Entities/ActSurvey/ActSurvey.cs
namespace Bars.GkhGji.Entities
{
    using System.Collections.Generic;

    using Bars.GkhGji.Enums;

    /// <summary>
    /// Акт обследования
    /// </summary>
    public class ActSurvey : DocumentGji
    {
        /// <summary>
        /// Обследованная площадь
        /// </summary>
        public virtual decimal? Area { get; set; }

        /// <summary>
        /// Квартира
        /// </summary>
        public virtual string Flat { get; set; }

        /// <summary>
        /// Причина
        /// </summary>
        public virtual string Reason { get; set; }

        /// <summary>
        /// Выводы по результату
        /// </summary>
        public virtual string Description { get; set; }

        /// <summary>
        /// Факт обследования
        /// </summary>
        public virtual SurveyResult FactSurveyed { get; set; }

        // ToDo ГЖИ следующие поля необходимо выпилить после перехода на правила

        /// <summary>
        /// Список жилых домов акта (Используется при создании объекта)
        /// </summary>
        public virtual List<long> RealityObjectsList { get; set; }

        /// <summary>
        /// Список родительских документов (Используется при создании объекта)
        /// </summary>
        public virtual List<long> ParentDocumentsList { get; set; }
    }
}